i18n/en.json

```diff
@@ -604,6 +604,42 @@
     "translation": "Error to update the current channel."
   },
   {
+    "id": "api.command_channel_rename.name",
+    "translation": "rename"
+  },
+  {
+    "id": "api.command_channel_rename.hint",
+    "translation": "[text]"
+  },
+  {
+    "id": "api.command_channel_rename.desc",
+    "translation": "Rename the channel."
```

Comment:
Remove period so it's consistent with other autocomplete descriptions

`Rename the channel`